package.json

```diff
@@ -27,7 +27,8 @@
   "scripts": {
     "build": "react-scripts build",
     "ci-lint": "prettier --check --loglevel error '{data_pipeline,.github,src,.vscode}/**/*.{json,css,scss,md}' && tsc --noEmit && eslint 'src/**/*.{js,jsx,ts,tsx}' --quiet",
-    "datapipe": "npm run strapped && npm run filter",
+    "combine": "node data_pipeline/combine.js",
+    "datapipe": "npm run strapped && npm run filter && npm run combine",
```

Comment:
do we want to just go ahead and remove the `strapped` references?

Reply:
I think we should keep it since that command actually goes and grabs data from strapped. Once we're completely independent, we can remove these commands.